fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/synonym/SynonymAnalyzer.java

```diff
@@ -50,13 +50,26 @@
  * The synonyms are not indexed to disk. So no index rebuilding is needed if the wordnet file gets updated.
  */
 public class SynonymAnalyzer extends StopwordAnalyzerBase {
-    private static final String WORDNET_FILE_NAME = "wn_s.pl";
+    @Nonnull
+    private final String name;
+
+    @Nonnull
+    public String getName() {
+        return name;
+    }
 
     @Nullable
-    private SynonymMap cachedSynonymMap = null;
+    public CharArraySet getStopwords() {
+        return stopwords;
+    }
 
     public SynonymAnalyzer(@Nullable CharArraySet stopwords) {
```

Comment:
Is this needed anymore? Is there a need, in general, to have a public constructor?